files/en-us/web/web_components/using_custom_elements/index.html

```diff
@@ -264,4 +264,4 @@ <h2 id="Transpilers_vs._classes">Transpilers vs. classes</h2>
 
 <h2 id="Libraries">Libraries</h2>
 
-<p>There are several libraries that are built on Web Components with the aim of increasing the level of abstraction when creating custom elements. Some of these libraries are <a href="https://www.fast.design/docs/fast-element/getting-started" rel="nofollow">FASTElement</a>, <a href="https://github.com/devpunks/snuggsi" rel="nofollow">snuggsi</a>, <a href="https://x-tag.github.io/" rel="nofollow">X-Tag</a>, <a href="http://slimjs.com/" rel="nofollow">Slim.js</a>, <a href="https://lit.dev/">Lit</a>, <a href="https://www.htmlelements.com/">Smart</a>, <a href="https://stenciljs.com">Stencil</a>, <a href="https://github.com/WebReflection/hyperHTML-Element">hyperHTML-Element</a>, and <a href="https://www.dataformsjs.com/">DataFormsJS</a>.</p>
+<p>There are several libraries that are built on Web Components with the aim of increasing the level of abstraction when creating custom elements. Some of these libraries are <a href="https://www.fast.design/docs/fast-element/getting-started" rel="nofollow">FASTElement</a>, <a href="https://github.com/devpunks/snuggsi" rel="nofollow">snuggsi</a>, <a href="https://x-tag.github.io/" rel="nofollow">X-Tag</a>, <a href="http://slimjs.com/" rel="nofollow">Slim.js</a>, <a href="https://lit.dev/">Lit</a>, <a href="https://www.htmlelements.com/">Smart</a>, <a href="https://stenciljs.com">Stencil</a>, <a href="https://github.com/WebReflection/hyperHTML-Element">hyperHTML-Element</a>, <a href="https://www.dataformsjs.com/">DataFormsJS</a>, and <a href="https://tmorin.github.io/ceb/">Custom-Element-Builder</a>.</p>
```

Comment:
It would be _great_ to fix the http -> https link here whilst we're at it!

```suggestion
<p>There are several libraries that are built on Web Components with the aim of increasing the level of abstraction when creating custom elements. Some of these libraries are <a href="https://www.fast.design/docs/fast-element/getting-started" rel="nofollow">FASTElement</a>, <a href="https://github.com/devpunks/snuggsi" rel="nofollow">snuggsi</a>, <a href="https://x-tag.github.io/" rel="nofollow">X-Tag</a>, <a href="https://slimjs.com/" rel="nofollow">Slim.js</a>, <a href="https://lit.dev/">Lit</a>, <a href="https://www.htmlelements.com/">Smart</a>, <a href="https://stenciljs.com">Stencil</a>, <a href="https://github.com/WebReflection/hyperHTML-Element">hyperHTML-Element</a>, <a href="https://www.dataformsjs.com/">DataFormsJS</a>, and <a href="https://tmorin.github.io/ceb/">Custom-Element-Builder</a>.</p>
```